routingv8/request.go

```diff
@@ -527,3 +529,54 @@ type RouteImportRequest struct {
 type RouteImportRequestBody struct {
 	Trace []GeoWaypoint `json:"trace"`
 }
+
+type Vehicle struct {
+	// Gross vehicle weight, including trailers and shipped goods when loaded at capacity, specified in kilograms.
+	// If unspecified, it will default to currentWeight.
+	// If neither parameter has a value specified, it will default to 0.
+	// Notes:
+	// - Supported in truck, bus, privateBus, car (Beta), taxi (Beta) transport modes.
+	// - Maximum weight for a car or taxi without a trailer is 4250 kg.
+	// - Maximum weight for a car or taxi with a trailer is 7550 kg.
+	GrossWeight int
+	// Default: 0. The number of trailers attached to the vehicle.
+	// Maximum value when used with transportMode=car or transportMode=taxi is 1.
+	// Limitations: Considered for route calculation when transportMode is one of (truck, bus, privateBus).
+	// Considered for route calculation for restrictions, but not for speed limits, when transportMode is car or taxi.
+	TrailerCount int
```

Comment:
Can we document the allowed range of 0-255? (Maybe it is worth using a `uint` instead?)